README.md

```diff
@@ -74,25 +75,36 @@ request for the code review
 👤 **Tsige Wossne**
 
 - GitHub: [@githubhandle](https://github.com/tsigewossne)
-- Twitter: [@twitterhandle] @Tsigewo
+- Twitter: [@twitterhandle] (https://twitter.com/TsigeWo)
+- Email: [@gmail] (tsigedejen@gmail.com)
 - LinkedIn: [LinkedIn](http://ca.linkedin.com/in/linkedintsigewossne)
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🔭 Future Features <a name="future-features"></a>
 
+- **Recent works**
+- **News**
+- **Full version portfolio**
 
-## 🔭 Future Features <a name="future-features"></a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- [ ] **Additional Tags**
-- [ ] **Simple buttons**
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to contact me.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please appriciate my work by following my social media profiles and provide your comments and suggestions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank the microverse team that stands me for this project including my morning team, code reviewers and my partners
+I would like to thank the microverse team that stands me for this project including my morning team, code reviewers and my partners.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- The link to your license file is broken, It currently leads to a 404 error page. I can see that you have a license file in your project but it's not correctly linked. I suggest you change this line from `This project is [MIT](./LICENSE) licensed.` to `This project is [MIT](./License.md) licensed.` to fix the issue. :+1: :smile: